pycbc/neutron_stars/ns_functions.py

```diff
@@ -174,11 +174,12 @@ def PG_ISSO_solver(chi,incl):
 
     return solution
 
+
 ##############################################################################
 # 2H 2-piecewise polytropic EOS, NS non-rotating equilibrium sequence        #
 # File format is: grav mass (Msun)   baryonic mass (Msun)    compactness     #
-#                                                                            #
-# Eventually, the function should call an NS sequence generator within LAL   #
+#                                                                            #       THIS FUNCTION IS BEING CALLED IN TMPLTBANK, does it need to be moved back?
+# Eventually, the function should call an NS sequence generator within LAL   #       or should I go and modify those files to reflect new location?
```

Comment:
Anywhere these functions are called from should be updated to point to the new location (and then remove this comment)

Reply:
Sorry about this, that was a very old comment I didn't spot before the PR, just fixed it.